plugins/modules/ndo_port_channel_interface.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_port_channel_interface
+short_description: Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.2 (NDO v4.4) and later.
+author:
+- Gaspard Micol (@gmicol)
+options:
+    template:
+        description:
+        - The name of the template.
+        - The template must be a tenant template.
+        type: str
+        required: true
+    port_channel_interface:
+        description:
+        - The name of the Port Channel Interface.
+        type: str
+        aliases: [ name, port_channel ]
+    port_channel_interface_uuid:
+        description:
+        - The uuid of the Port Channel Interface.
+        - This parameter is required when the O(port_channel_interface) needs to be updated.
+        type: str
+        aliases: [ uuid, port_channel_uuid ]
+    description:
+        description:
+        - The description of the Port Channel Interface.
+        type: str
+    node:
+        description:
+        - The node ID.
+        type: str
+    member_interfaces:
+        description:
+        - The list of used Interface IDs.
+        type: list
+        elements: str
+        aliases: [ members ]
+    interface_policy:
+        description:
+        - The name of the Port Channel Interface Setting Policy.
+        type: str
+        aliases: [ policy, pc_policy ]
+    interface_policy_uuid:
+        description:
+        - The UUID of the Port Channel Interface Setting Policy.
+        type: str
+        aliases: [ policy_uuid, pc_policy_uuid ]
+    interface_descriptions:
+        description:
+        - The list of descriptions for each interface.
+        type: list
+        elements: dict
+        suboptions:
+            interface_id:
+                description:
+                - The interface ID to which attach a description.
```

Comment:
sentence seems off, could you rephrase
```suggestion
                - The interface ID to which attach a description.
```

Reply:
will do